phases/10-llms-from-scratch/01-tokenizers/code/main.py

```
@@ -0,0 +1,227 @@
+    print("=" * 60)
+
+    test_texts = [
+        corpus,
+        "The quick brown fox jumps over the lazy dog.",
+        "Machine learning is a subset of artificial intelligence.",
+        "Python is the most popular language for data science.",
+    ]
+
+    vocabulary_stats(tokenizer, test_texts)
+
+    print(f"\nCompression ratios:")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)